pkg/kubelet/runonce.go

```diff
@@ -41,6 +42,18 @@ type RunPodResult struct {
 
 // RunOnce polls from one configuration update and run the associated pods.
 func (kl *Kubelet) RunOnce(updates <-chan kubetypes.PodUpdate) ([]RunPodResult, error) {
+	// Setup filesystem directories.
+	if err := kl.setupDataDirs(); err != nil {
+		return nil, err
+	}
+
+	// If the container logs directory does not exist, create it.
```

Comment:
This is pasted from elsewhere in kubelet.go, extract method please?  In a separate PR if this has merged by the time you see this -- I won't rob you of LGTM.